gateway/ln-gateway/Cargo.toml

```diff
@@ -50,6 +50,7 @@ fedimint-mint-client = { version = "=0.4.0-alpha", path = "../../modules/fedimin
 fedimint-wallet-client = { version = "=0.4.0-alpha", path = "../../modules/fedimint-wallet-client" }
 fedimint-lnv2-client = { path = "../../modules/fedimint-lnv2-client" }
 fedimint-lnv2-common = { path = "../../modules/fedimint-lnv2-common" }
+fedimint-lnv2-server = { path = "../../modules/fedimint-lnv2-server" }
```

Comment:
Why does the gateway need the server module as a dependency? The gateway is a client, something doesn't feel right

Reply:
This should be a dev dependency

Reply:
Done